src/main/kotlin/me/partlysanestudios/partlysaneskies/features/economy/auctionhousemenu/AuctionHouseGui.kt

```diff
@@ -194,9 +194,6 @@ class AuctionHouseGui(defaultAuctionInventory: IInventory) : WindowScreen(Elemen
                 return
             }
 
-            if (DebugKey.isDebugMode()) {
-                return
-            }
```

Comment:
This still wouldn't fix the problem right?

Reply:
Idk cant test

Reply:
Yeah but you can logic

Reply:
The logic alr says that it shouldnt be needed to remove tho
So logic doesnt really work

Reply:
Yeah so logic says that that line of code is not the problem

Reply:
yeah but the user said that it didnt work without it

Reply:
Just test it pretty please?

Reply:
I can't right now

Reply:
Okay

Reply:
Could we leave it

Reply:
wdym leave it

Reply:
This check